### PR TITLE
feat: Adapter, defi.money

### PR DESCRIPTION
**Pull request by 0xpeluche**
---
##### Name (to be shown on DefiLlama):
defi.money

##### Twitter Link:
https://x.com/defidotmoney

##### Website Link:
https://defi.money/

##### Chain:
arbitrum, optimism

##### Category (full list at https://defillama.com/categories) *Please choose only one:
CDP & Yield (there is 2 different products split in 2 different folders)

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/defi-money exports TVL: 
        
 
 
    optimism                  840.82 k
    arbitrum                  781.63 k
    
    total                    1.62 M

**Comment by llamatester** [comment]
The adapter at projects/defi-money-cdp exports TVL: 
        
 
 
    optimism                  836.15 k
    arbitrum                  777.78 k
    
    total                    1.61 M

**Comment by llamatester** [comment]
The adapter at projects/defi-money-earn exports TVL: 
        
 
 
    arbitrum                  975.75 k
    optimism                  891.71 k
    
    total                    1.87 M

File: projects/defi-money-cdp/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const getTvl = async (api, controller, viewer) => {
+  const vaults = await getVaults(api, controller);
+  await getVaultsBalances(api, viewer, vaults);
+};
+
+Object.keys(config).forEach((chain) => {
+  const { controller, viewer } = config[chain];
+  module.exports[chain] = {
+    methodology: "TVL corresponds to the collateral deposited in the markets",
```
Comment (g1nt0ki) [inline projects/defi-money-cdp/index.js:49]: we set the methodology at the top level, not per chain

I am not a fan of this pattern of splitting into micro functions as I have to read bottom up, feel like it is easier if everything is in a single function so one can read it like a script (just a suggestion)

File: projects/defi-money-earn/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "0xdE718A791226c93B53C77D60E5D4693C05a31422", // USDC/MONEY
+      "0xE3763d545707F435e21eeBbe75070495c806B744", // DAI/MONEY
+      "0x07aDF588508b923B8eA0389d27b61b9CB8a197Cb", // FRAX/MONEY
+    ],
+  },
+};
+
+const getLPTokens = async (api, stakingContracts) => {
+  const stakeTokens = await api.multiCall({
+    calls: stakingContracts.map((address) => ({ target: address })),
```
Comment (g1nt0ki) [inline projects/defi-money-earn/index.js:32]: this wrapping is not needed

**Comment by llamatester** [comment]
The adapter at projects/defi-money-earn exports TVL: 
        
 
 
    arbitrum                  978.30 k
    optimism                  894.23 k
    
    total                    1.87 M

**Comment by llamatester** [comment]
The adapter at projects/defi-money exports TVL: 
        
 
 
    pool2                     1.78 M
    arbitrum-pool2            942.13 k
    optimism                  850.80 k
    optimism-pool2            834.24 k
    arbitrum                  792.33 k
    
    total                    1.64 M

**Comment by llamatester** [comment]
The adapter at projects/defi-money exports TVL: 
        
 
 
    pool2                     1.78 M
    arbitrum-pool2            942.13 k
    optimism                  850.80 k
    optimism-pool2            834.11 k
    arbitrum                  792.33 k
    
    total                    1.64 M

**Pull request merged**